lib/oai/provider/metadata_format/hyku_dublin_core.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module OAI
+  module Provider
+    module MetadataFormat
+      class HykuDublinCore < OAI::Provider::Metadata::Format
+        def initialize
+          @prefix = 'oai_hyku'
+          @schema = 'http://dublincore.org/schemas/xmls/qdc/dcterms.xsd'
+          @namespace = 'http://purl.org/dc/terms/'
+          @element_namespace = 'hyku'
```

Comment:
Are these three values appropriate / correct?

Reply:
this is fine. it's not exactly "proper" but its the right thing to do in order to map correctly in other systems.